executorCommon/src/main/scala/dx/executor/WorkflowExecutor.scala

```diff
@@ -721,7 +721,7 @@ abstract class WorkflowExecutor[B <: Block[B]](jobMeta: JobMeta, separateOutputs
     def launch(): Map[DxName, ParameterLink] = {
       val outputs: Map[DxName, ParameterLink] = block.kind match {
         case BlockKind.CallWithSubexpressions | BlockKind.CallFragment =>
-          launchCall(block.index)
+          launchBlockCall(block.index)
```

Comment:
Rename this function here to fit the three launch scenarios: block, conditional, scatter.

All three would later call `launchCall` function to call the applet and launch a job.